integrations/cassandra/vendor/vendor.json

```diff
@@ -0,0 +1,63 @@
+{
+	"comment": "",
+	"ignore": "test",
+	"package": [
+		{
+			"checksumSHA1": "nW9vfRK+2x+DkbG39uSmI/WJooc=",
+			"path": "github.com/newrelic/infra-integrations-sdk/args",
+			"revision": "75643537def5cad61b5966440fd80b61609c68e3",
+			"revisionTime": "2018-05-02T09:26:36Z"
```

Comment:
I think that we should fetch specific version. However, I know that we still didn't release version v3.0.0, we have v3.0.1-beta. And not sure how we could manage it.